<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RedminePanel</class>
 <widget class="QDialog" name="RedminePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="3" column="1">
      <widget class="QCheckBox" name="checkBox_statusCheck">
       <property name="text">
        <string>审核&amp;评审</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_assignedAdd">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="comboBox_assignedTo">
       <item>
        <property name="text">
         <string>软件一部</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>软件二部</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>软件三部</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>测试部</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>技术支持部</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>外贸支持</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>全部</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="checkBox_statusOver">
       <property name="text">
        <string>已完成</string>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QPushButton" name="pushButton_renew">
       <property name="text">
        <string>更新数据库</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="checkBox_statusWaitcheck">
       <property name="text">
        <string>待验证</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="pushButton_assignedDel">
       <property name="text">
        <string>删除选中</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" rowspan="4" colspan="2">
      <widget class="QListWidget" name="listWidget_partMembers"/>
     </item>
     <item row="2" column="4" rowspan="4" colspan="2">
      <widget class="QListWidget" name="listWidget_assignedTo"/>
     </item>
     <item row="2" column="6">
      <widget class="QPushButton" name="pushButton_set">
       <property name="text">
        <string>查询设置</string>
       </property>
      </widget>
     </item>
     <item row="4" column="6">
      <widget class="QPushButton" name="pushButton_count">
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="pushButton_assignedClear">
       <property name="text">
        <string>清除</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>任务类型</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>任务状态</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="checkBox_statusNew">
       <property name="text">
        <string>新建</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="checkBox_taskTest">
       <property name="text">
        <string>测试</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="checkBox_taskHard">
       <property name="text">
        <string>硬件任务</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="checkBox_taskBug">
       <property name="text">
        <string>BUG</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="checkBox_taskTask">
       <property name="text">
        <string>任务</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="checkBox_statusWorking">
       <property name="text">
        <string>进行中</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="7">
      <widget class="QTableWidget" name="tableWidget_taskList"/>
     </item>
     <item row="0" column="2" colspan="4">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>指派给</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>查询</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="7">
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>任务清单</string>
       </property>
      </widget>
     </item>
     <item row="5" column="6">
      <widget class="QPushButton" name="pushButton_export">
       <property name="text">
        <string>导出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
